user.css

```diff
@@ -1,431 +1,113 @@
+    --text-main-200: #000 !important;
+    --text-main-300: #000 !important;
+    --text-main-400: #4b4b4b !important;
+    --text-hover: #000 !important;
+    --text-white: #fff !important;
 
-#footer,
-.basicContext,
-#lychee_footer {
-    background: var(--footer-background);
-}
+    --bg-50: #fff !important;
+    --bg-100: #fff !important;
+    --bg-200: #fff !important;
+    --bg-300: #fff !important;
+    --bg-400: #fff !important;
+    --bg-500: #f1f1f1 !important;
+    --bg-600: #f1f1f1 !important;
+    --bg-700: #fff !important;
+    --bg-800: #fff !important;
+    --bg-900: #fff !important;
+    --bg-950: #fff !important;
 
-.basicContext {
-    color: var(--context-color);
-    border: var(--context-border);
-    box-shadow: var(--context-box-shadow);
-}
+    --danger: #dc2626;
+    --danger-dark: #b91c1c;
+    --danger-darker: #991b1b;
 
-.basicContext__data .iconic {
-    filter: invert(100%);
-}
+    --success: #22c55e;
 
-.basicContext__data {
-    color: var(--context-data-color);
-}
+    --create: #16a34a;
+    --create-dark: #15803d;
 
-#imageview.full {
-    background: var(--background-color);
-    backdrop-filter: blur(20px);
-    -webkit-backdrop-filter: blur(20px);
-}
+    --warning: #fb923c;
+    --warning-dark: #ea580c;
 
-#imageview {
-    background: var(--imageview-background);
+    --thumb-shadow-color: rgba(0, 0, 0, 0.05);
+    --overlay-bg: rgba(255, 255, 255, 0.8);
+    --overlay-shadow-color: rgba(0, 0, 0, 0.1);
+    --preview-overlay-bg: rgba(255, 255, 255, 0.7);
 }
 
-#imageview #image_overlay {
-    color: var(--imageview-overlay-color);
-    text-shadow: var(--imageview-overlay-text-shadow);
+#lychee_toolbar_container {
+    border-bottom-color: #ececec !important;
 }
 
-#image_overlay > p > a > span {
-    filter: var(--imageview-overlay-span-filter);
+.album-thumb .overlay {
+    text-shadow: none !important;
 }
 
-.leftMenu,
-#lychee_left_menu_container {
-    filter: var(--leftmenu-filter);
-    transition: var(--leftmenu-transition);
+.album-thumb .overlay h1 {
+    font-weight: 400 !important;
+    font-size: 1rem !important;
 }
 
-.leftMenu.leftMenu__visible .closetxt {
-    display: none;
+.album-thumb .overlay {
+    background: var(--preview-overlay-bg) !important;
 }
 
-.leftMenu a.linkMenu,
-.leftMenu.leftMenu__visible .closebtn {
-    cursor: pointer;
+.thumbimg {
+    box-shadow: 2px 2px 16px 1px var(--thumb-shadow-color) !important;
 }
 
-#imageview,
-.sidebar,
-#lychee_sidebar {
-    backdrop-filter: blur(15px);
-    -webkit-backdrop-filter: blur(15px);
+.thumbimg img:not([data-src]) {
+    filter: invert(1) !important;
 }
 
-.sidebar,
-#lychee_sidebar {
-    background: var(--sidebar-background);
-    border-top: var(--sidebar-border-top);
+.text-text-main-200 {
+    color: var(--text-main-0) !important;
 }
 
-.sidebar h1,
-.sidebar table tr td,
-#lychee_sidebar h1,
-#lychee_sidebar table tr td {
-    color: var(--sidebar-text-color) !important;
+.text-text-main-300 {
```

Comment:
Same as above.

Reply:
You are right, thanks, I've removed extra classes